charts/opentelemetry-operator/templates/clusterrolebinding.yaml

```diff
@@ -5,14 +5,14 @@ metadata:
   labels:
     {{- include "opentelemetry-operator.labels" . | nindent 4 }}
     app.kubernetes.io/component: controller-manager
-  name: {{ template "opentelemetry-operator.name" . }}-manager-rolebinding
+  name: {{ template "opentelemetry-operator.fullname" . }}-manager
```

Comment:
@sfenman this means that both roles and rolebindings will have the same resource name. Maybe we should add `rb` suffix as a differentiator? Same for other resources with duplicate names ?

Reply:
I removed it because they are already differentiated by their `Kind` attribute. For sure I can add an `rb` suffix if you think it is confusing though.

Reply:
@TylerHelmuth @dmitryax thoughts?

Reply:
I think `Kind` is enough.  The collector chart doesn't do any extra naming for clusterrole vs clusterrolebinding.